### DISCUSSION TITLE
Heist Control 2.29.4

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Heist-Control-v2#56] [General]
## Feel free to give suggestions or report bugs.

**Comment by hinatiki-main** [comment]
Hello. The script you are developing is very nice. However, only the Instant Finish feature does not work well in my environment. Sometimes it succeeds, but the success rate is about 30%. Especially on Doomsday heist, whenever I use it, the screen turns red and the mission fails. In Classic heist, when I do it, there is no reaction or the map disappears and nothing happens. Others seem to think that if the ping value of the person I am playing with is too high it will not work. or has R* patched this? what is your opinion on this matter?

**Reply by jhowkNx** [reply]
Usually these problems occur because of latency between you and the other players you are with, this is all based on R* servers.
Unfortunately there is no control over this, I can only suggest that if you have too many problems in a row, try another heist.

**Comment by mustanggamer** [comment]
i am having problems with getting the scrip working i'm also not getting scrip features

**Reply by jhowkNx** [reply]
I assure you that this is not a problem of the Script, make sure you always use the latest version. If the options did not appear in the "Script Features" it is because it was not able to load correctly.

**Comment by Foltzy2015** [comment]
I can't get the heist to work. It sets the boards up but is not giving out the 3.5M that it did in the past. I am not sure why this isn't working. I also have to pick up cash like you normally would to even have any chance of getting money... Did I miss something here? Am I just not doing this correctly?

**Comment by 299km1h** [comment]
Hello, first of all, thank you for allowing me to use such a good script. I want to ask, after the game is updated, it seems that the unlimited backpack function of Perrick Island in the robbery control script can't be used. Is this the one I used incorrectly?
